### DISCUSSION TITLE
Build error in Cygwin

### DISCUSSION THREAD

**Discussion by guliteb** [ipxe/wimboot#35] [Q&A]
I am new to ipxe. I setup Cygwin under windows. Simply run 'make' to build the code under src. Following are part of the error. I searched around and cannot find similar issue. Any help is appreciated.

cc -Os -ffreestanding -Wall -W -Werror -nostdinc -I. -fshort-wchar -DVERSION="\"v2.7.4\"" -DSBAT_GENERATION="\"1\"" -fstack-protector-strong  -mstack-protector-guard=global -fno-dwarf2-cfi-asm -fno-exceptions  -fno-unwind-tables -fno-asynchronous-unwind-tables  -Wno-address-of-packed-member -fno-lto  -include compiler.h -m64 -mno-red-zone -fpie -DASSEMBLY -Ui386 -E prefix.S -o prefix.x86_64.s
as --64 x86_64.i prefix.x86_64.s -o prefix.x86_64.o
prefix.S: Assembler messages:
prefix.S:75: Error: junk at end of line, first unrecognized character is `,'
prefix.S:85: Warning: .size pseudo-op used outside of .def/.endef: ignored.
prefix.S:85: Error: junk at end of line, first unrecognized character is `m'
prefix.S:99: Warning: .size pseudo-op used outside of .def/.endef: ignored.

**Comment by NiKiZe** [comment]
Cygwin is not a valid build environment for iPXE, but it should work in WSL and WSL2 environments.
